src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -493,8 +493,9 @@ static int do_map()
     }
 
     if (forker.is_parent()) {
+      global_init_postfork_start(g_ceph_context);
```

Comment:
For me both versions look working. Anyway, I think the current initialization code in do_map already a bit confusing and it would be nice to try clean it up (as a separate PR, I could try this), so now any version that just fixes the issue could be ok.